tw-tkms-starter/src/test/resources/application.yml

```diff
@@ -101,7 +101,7 @@ tw-tkms:
   database-dialect: POSTGRES
   earliest-visible-messages:
     enabled: true
-    look-back-period: 10
+    look-back-period: 10s
```

Comment:
It was always intended to be 10s, but it was actually set as 10ms. Added a startup validation for it.